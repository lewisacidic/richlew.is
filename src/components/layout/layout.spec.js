import React from 'react'
import { render } from 'react-testing-library'

import Layout from './layout'

describe('Layout', () => {
  it('works', () => {
    const { container } = render(<Layout />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
it('works', () => {
  expect(true).toBe(true)
})
