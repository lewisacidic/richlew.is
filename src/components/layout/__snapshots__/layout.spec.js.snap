// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout works 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  position: absolute;
  background: linear-gradient(120deg,#8ddad5,#00cdac);
}

.c2 {
  display: inline-block;
  padding: 1em;
  margin-bottom: 0;
}

.c1 {
  list-style-type: none;
  text-align: center;
  margin-bottom: 0;
}

.c3 {
  background: white;
  max-width: 960px;
  margin: auto;
  padding: 3rem;
  border-radius: 3rem;
}

.c3 > :last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  <ul
    class="c1"
  >
    <li
      class="c2"
    >
      <a
        aria-current="page"
        class=""
        href="/"
        style="text-decoration: none; color: white;"
      >
        home
      </a>
    </li>
    <li
      class="c2"
    >
      <a
        href="/contact"
        style="text-decoration: none; color: white;"
      >
        contact
      </a>
    </li>
  </ul>
  <div
    class="c3"
  />
</div>
`;
